src/Component/Header.jsx

```diff
@@ -0,0 +1,22 @@
+import { NavLink } from "react-router-dom";
+import Logo from "C:/Users/ouran/Desktop/tmp/7-Sprint-Mission/src/images/logo/panda-market-pandalogo.svg";
+import "../style/Header.css";
+
+function Header() {
+    return (
+        <header>
+            <nav>
+                <NavLink to="/"><img src={Logo} alt="판다마켓로고" className="headerLogo"></img></NavLink>
```

Comment:
img 태그는 children 요소가 없으므로 바로 close 해주셔도 될 것 같습니다.
```suggestion
                <NavLink to="/"><img src={Logo} alt="판다마켓로고" className="headerLogo" /></NavLink>
```